codebase/config/sync/field.storage.media.field_media_of.yml

```diff
@@ -6,8 +6,12 @@ dependencies:
     module:
       - islandora_core_feature
   module:
+    - field_permissions
```

Comment:
Do you know what this does?  must be part of the media_of being required change.

Reply:
Unfortunately, I do not.